### DISCUSSION TITLE
Tooltip -> meeting types

### DISCUSSION THREAD

**Discussion by na-nederland** [code4recovery/12-step-meeting-list#1319] [Q&A] [answered]
We do get some questions for explanation of meeting types. Not only from newcomers. We can add a page to the website explaining the types, another more convenient solution would be tooltips. 

Thanks in advance for any suggestions! 🙏

**Comment by joshreisner** [comment, marked as answer]
hi we have a thing for this called a "type description" - you can read more about how to customize those here: https://wordpress.org/plugins/12-step-meeting-list/#can%20i%20update%20the%20type%20descriptions%3F

we have a few built in descriptions for AA, but probably not for NA

we can talk about what the right UI is for these - tooltips typically don't work well on mobile, where we see most of our users, but there are probably improvements we can make in how we show them

**Reply by na-nederland** [reply]
Thanks! We will have a look 🙏
